website/source/docs/commands/debug.html.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: "docs"
+page_title: "debug - Command"
+sidebar_title: "<code>debug</code>"
+sidebar_current: "docs-commands-debug"
+description: |-
+  The "debug" command monitors a Vault server, probing information about
+  it for a certain duration.
+---
+
+# debug
+
+The `debug` command starts a process that monitors a Vault server, probing
+information about it for a certain duration.
+
+Gathering information about the state of the Vault cluster often requires the
+operator to access all necessary information via various API calls and terminal
+commands. The `debug` command aims to provide a simple workflow that produces a
+consistent output to help operators retrieve and share information about the
+server in question.
+
+The `debug` sub-command honors the same the variables that the base command
```

Comment:
In the first sentence of the documentation we refer to it as the "the `debug` command", but here we refer to it as "the `debug` sub-command". Do we consider these to be the same, command and sub-command?

Reply:
The docs for other top-level commands refer them simply as commands so I've updated it here to be likewise.